faux-mgs/src/usart.rs

```diff
@@ -245,6 +263,13 @@ impl StdinOutBuf {
                         self.buf.push(c);
                     }
                 }
+                CTRL_BACKSLASH => {
+                    if self.in_prefix {
+                        return IngestResult::Break;
```

Comment:
This early return will cause any characters remaining in `buf` to be silently discarded. (The same is true on the early exit return, but we're exiting anyway.) Maybe we should have a `mut result` that we modify in this loop, and always run it to completion before returning?

Reply:
That makes sense to me, done in 4aba91d7d4690abad4c1ada04610570f22cc1886